predicators/args.py

```diff
@@ -41,4 +41,7 @@ def create_arg_parser(env_required: bool = True,
                         const=logging.DEBUG,
                         default=logging.INFO)
     parser.add_argument("--crash_on_failure", action="store_true")
+    # Used when running train_refinement_estimator.py from the command line
```

Comment:
we should either:
1. make this change inside of `train_refinement_estimator.py`, for example, by calling this function to get the initial `parser` and then called `add_argument(...)` on it after it's returned
2. change these flag names to make it clear that they're about refinement estimator training only

or both. my preference is both!

Reply:
Change the utils.parse_args to use a helper function that takes in a parser as input.